src/test/java/com/google/api/codegen/InflectorTest.java

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen;
+
+import com.google.common.truth.Truth;
+import org.junit.Test;
+
+public class InflectorTest {
+
+  @Test
+  public void testSingularize() {
+    Truth.assertThat(Inflector.singularize("scarves")).isEqualTo("scarf");
+    Truth.assertThat(Inflector.singularize("halves")).isEqualTo("half");
+    Truth.assertThat(Inflector.singularize("cars")).isEqualTo("car");
+    Truth.assertThat(Inflector.singularize("fuzzes")).isEqualTo("fuzz");
+    Truth.assertThat(Inflector.singularize("sneezes")).isEqualTo("sneeze");
+    Truth.assertThat(Inflector.singularize("blesses")).isEqualTo("bless");
+    Truth.assertThat(Inflector.singularize("scarves")).isEqualTo("scarf");
```

Comment:
Looks like this case is written twice (another on L24)?
For this kind of tests, I like sorting alphabetically but I won't push on that :P

Reply:
Ah sorting alphabetically would have caught the dup! Done.